humanitec.js

```diff
@@ -38,14 +38,13 @@ module.exports = function(token, orgId, apiHost) {
   }
 
   /**
-   * Notifies Humanitec that a build has completed
-   * @param {string} imageName - The name of the image to be added to Huamnitec.
-   * @param {Payload} payload - Details about the image.
+   * Notifies Humanitec that a version has been added
+   * @param {Payload} payload - Details about the artefact version.
    * @return {Promise} - A promise which resolves to true if successful, false otherwise.
    */
-  function addNewBuild(imageName, payload) {
+  function addNewVersion(payload) {
     return fetch(
-      `https://${apiHost}/orgs/${orgId}/modules/${imageName}/builds`, {
+      `https://${apiHost}/orgs/${orgId}/artefact-versions`, {
         method: 'POST',
         headers: {
           'Authorization': `Bearer ${token}`,
```

Comment:
Unrelated, but could we add a `User-Agent` here to see which calls are coming from this action? E.g. `gh-action-build-push-to-humanitec/version`?